File: compare/language loop speed/Loop.java
public class Loop {
    public static int ALOOPTIME=10000;
    public static  int LOOP=10000;
    public  static void loop10k(){
        int i = 0;
        while(i<ALOOPTIME)i++;
    }
    public static void main(String args[]) {
        for(int i =0;i <LOOP;i++)
            loop10k();
        System.out.print("loop " + LOOP + "*" + ALOOPTIME + " times\n");
      
   }
}

File: compare/language loop speed/loop.lua
function add(a,b)
    return a+b
end
alooptime=10000
function loop10k()
    for i=0,alooptime,1 do
    end
end

i=0
LOOP1=10000
while(i<LOOP1)
do
i=i+1
loop10k()
end
counter=LOOP1*alooptime
print("loop ",LOOP1,"*",alooptime," times")
--print( add(3,4))
